src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -80,16 +77,13 @@ public static int LastIndexOfAny(ref byte searchSpace, int searchSpaceLength, re
             if (valueLength == 0)
                 return 0;  // A zero-length sequence is always treated as "found" at the start of the search space.
 
-            int index = -1;
+            int index = int.MinValue;
             for (int i = 0; i < valueLength; i++)
             {
                 var tempIndex = LastIndexOf(ref searchSpace, Unsafe.Add(ref value, i), searchSpaceLength);
-                if (tempIndex != -1)
-                {
-                    index = (index == -1 || index < tempIndex) ? tempIndex : index;
-                }
+                index = (tempIndex == -1 || index > tempIndex) ? index : tempIndex;
```

Comment:
The check for `tempIndex == -1` can be omitted here I think.

Reply:
Due the change above (line 80) yes, because `index` is initialized to -1.